src/index.js

```diff
@@ -195,7 +195,7 @@ const createWindow = () => {
 
   app.on('web-contents-created', (e, contents) => {
     if (contents.getType() === 'webview') {
-      contents.on('new-window', (event) => {
+      contents.on('new-window', (event) => {        
```

Comment:
@mahadevans87 : Please remove the extra spaces at the end. 
```suggestion
      contents.on('new-window', (event) => {
```